docs/reference/items.md

```diff
@@ -348,62 +633,170 @@ mutation {
 }
 ```
 
----
+</template>
+<template #sdk>
 
-## Update Multiple Items
+```js
+import { createDirectus } from '@directus/sdk';
+import { rest, updateItem } from '@directus/sdk/rest';
 
-Update multiple items at the same time.
+const client = createDirectus('https://directus.example.com').with(rest());
 
-### Query Parameters
+const result = await client.request(
+	updateItem('articles', '5', {
+		title: 'What is Directus and how it can help you build your next app!?',
+	})
+);
 
-Supports all [global query parameters](/reference/query).
+console.log(result);
+```
 
-### Request Body
+</template>
 
-Object containing `data` for the values to set, and either `keys` or `query` to select what items to update.
+</SnippetToggler>
 
-### Returns
+## Update Singleton
 
-Returns the [item objects](#the-item-object) for the updated items.
+Update a singleton item
 
-#### Singleton
+### Request
 
-If your collection is a singleton, this endpoint will act the same as the [Update an Item](#update-an-item) endpoint.
+```js
+import { createDirectus } from '@directus/sdk';
+import { rest, updateSingleton } from '@directus/sdk/rest';
+const client = createDirectus('directus_project_url').with(rest())
 
-### REST API
+const result = await client.request(updateSingleton('collection_name', {
+    'item_field' : 'value'
+}))
 
-```
-PATCH /items/:collection
+console.log(result);
 ```
 
-##### Example
+::: tip Info
 
+This endpoint is only supported by the SDK.
+
+:::
```

Comment:
Same as above this not an actual endpoint but identical to updating a single item
```suggestion
```

Reply:
Addressed in https://github.com/directus/directus/pull/19110/commits/488bf7050a859aa2f37193bfde7afc344cab7593